src/Error/Debug/dumpHeader.html

```diff
@@ -211,6 +225,27 @@
   }
 }
 
+function attachCopyButton(nodes) {
+  nodes.forEach(function (container) {
+    var copy = doc.createElement('a');
+    copy.classList.add('cake-dbg-copy');
+    copy.setAttribute('href', '#')
+    copy.setAttribute('title', 'Copy contents of debug output');
+    copy.appendChild(doc.createTextNode('Copy'));
+
+    // Add copy behavior
+    copy.addEventListener('click', function (event) {
+      event.preventDefault();
+      event.stopPropagation();
+
+      // Chop off last 4 to exclude copy button text.
+      navigator.clipboard.writeText(container.innerText.substring(0, container.innerText.length - 4));
```

Comment:
Personally I'm good with using the Clipboard API, it's way faster than the textarea hack, however in Firefox for example the API is only available in secure context's, ie it requires using HTTPS or `127.0.0.1`/`localhost` for serving the content. 

Would the people that use non localhost domains without SSL be of any concern?

Reply:
They aren't a concern I have. If the clipboard API doesn't work I don't think we need to add elaborate workarounds for this kind of feature.

Reply:
OK. But maybe an alert in case the Clipboard API isn't available (or writing fails) would be helpful, so that people aren't just left with nothing happening.